ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -10,10 +12,17 @@ export class MenuComponent implements OnInit {
   events: string[] = [];
   opened: boolean;
   buttonsArray = [];
-  constructor() { }
+  message: boolean;
+  constructor(private data: DataService) { }
+
 
   ngOnInit() {
     this.buttonsArray = BUTTONS;
+    this.data.currentMessage.subscribe(message => this.message = message);
```

Comment:
You should not receive data here. Move this logic to modal component

Reply:
this line is no longer